azure-pipelines-dependency-graph-testing.yml

```diff
@@ -245,13 +246,23 @@ stages:
 
           - script: |
               # Install dependencies
-              pip install pytest-cov pytest-icdiff pytest-mock pytest-azurepipelines invoke
+              pip install -r requirements-dev-test.txt
 
               # Run unit-tests
               invoke tests --ci --cloud  --timeout=3.0
 
             displayName: 'Unit Tests'
 
+          - script: |
+              pip install coveralls
+              export CIRCLE_BRANCH=$BUILD_SOURCEBRANCH
+              coveralls
+            displayName: 'coveralls'
+            env:
+              CIRCLECI: 1
+              CIRCLE_BUILD_NUM: $(Build.BuildNumber)
+              COVERALLS_REPO_TOKEN: $(coveralls)
```

Comment:
This may not be an issue on this PR, but for subsequent PRs ... our unit-test coverage numbers appear to have already dropped below the `40%` threshold since there was nothing to enforce it.

https://coveralls.io/builds/52134850